include/picongpu/fields/currentDeposition/Esirkepov/Esirkepov.hpp

```diff
@@ -114,23 +119,23 @@ namespace picongpu
                 using namespace cursor::tools;
                 cptCurrent1D(
                     acc,
-                    DataSpace<simDim>(leaveCell.y(), leaveCell.z(), leaveCell.x()),
+                    DataSpace<simDim>(status.y(), status.z(), status.x()),
                     twistVectorFieldAxes<pmacc::math::CT::Int<1, 2, 0>>(cursorJ),
                     rotateOrigin<1, 2, 0>(line),
                     cellSize.x());
                 cptCurrent1D(
                     acc,
-                    DataSpace<simDim>(leaveCell.z(), leaveCell.x(), leaveCell.y()),
+                    DataSpace<simDim>(status.z(), status.x(), status.y()),
                     twistVectorFieldAxes<pmacc::math::CT::Int<2, 0, 1>>(cursorJ),
                     rotateOrigin<2, 0, 1>(line),
                     cellSize.y());
-                cptCurrent1D(acc, leaveCell, cursorJ, line, cellSize.z());
+                cptCurrent1D(acc, status, cursorJ, line, cellSize.z());
             }
 
             /**
              * deposites current in z-direction
```

Comment:
I think it would be useful to extend this a little bit, that here x/y/z are not necessarily the same as in main PIConGPU cooridnate system, but it's after rotation